boards/nxp/tropic-community/nuttx-config/scripts/itcm_static_functions.ld

```diff
@@ -115,18 +115,29 @@
 *(.text.work_thread)
 *(.text.work_queue)
 
-/* Flash Storage */
-*(.text.imxrt_flexspi_transfer_blocking)
-*(.text.imxrt_flexspi_transfer_blocking_private)
-*(.text.imxrt_flexspi_write_blocking)
-*(.text.imxrt_flexspi_read_blocking)
-*(.text.imxrt_flexspi_check_and_clear_error)
-*(.text.imxrt_flexspi_get_bus_idle_status)
-*(.text.imxrt_flexspi_configure_prefetch)
-*(.text.imxrt_flexspi_configure_prefetch_private)
-*(.text.imxrt_flexspi_storage_write_enable)
-*(.text.imxrt_flexspi_storage_wait_bus_busy)
-*(.text.imxrt_flexspi_storage_read_status)
-*(.text.imxrt_flexspi_storage_erase)
-*(.text.imxrt_flexspi_storage_bwrite)
-*(.text.imxrt_flexspi_storage_page_program)
+/* BMI088 */
+*(.text._ZN5Bosch6BMI08813Accelerometer20BMI088_Accelerometer12RegisterReadENS1_8RegisterE)
+*(.text._ZN5Bosch6BMI08813Accelerometer20BMI088_Accelerometer13FIFOReadCountEv)
+*(.text._ZN5Bosch6BMI08813Accelerometer20BMI088_Accelerometer8FIFOReadERKyh)
+*(.text._ZN5Bosch6BMI08813Accelerometer20BMI088_Accelerometer13RegisterCheckERKNS2_17register_config_tE)
+*(.text._ZN5Bosch6BMI08813Accelerometer20BMI088_Accelerometer13RegisterWriteENS1_8RegisterEh)
+*(.text._ZN5Bosch6BMI08813Accelerometer20BMI088_Accelerometer17UpdateTemperatureEv)
+*(.text._ZN5Bosch6BMI08813Accelerometer20BMI088_Accelerometer23RegisterSetAndClearBitsENS1_8RegisterEhh)
+*(.text._ZN5Bosch6BMI08813Accelerometer20BMI088_Accelerometer26DataReadyInterruptCallbackEiPvS3_)
+*(.text._ZN5Bosch6BMI08813Accelerometer20BMI088_Accelerometer7RunImplEv)
+*(.text._ZN5Bosch6BMI08813Accelerometer20BMI088_Accelerometer9DataReadyEv)
+*(.text._ZN5Bosch6BMI08813Accelerometer20BMI088_AccelerometerD0Ev)
+*(.text._ZN5Bosch6BMI08813Accelerometer20BMI088_AccelerometerD1Ev)
+*(.text._ZN5Bosch6BMI08813Accelerometer20BMI088_AccelerometerD2Ev)
+*(.text._ZN5Bosch6BMI0889Gyroscope16BMI088_Gyroscope12RegisterReadENS1_8RegisterE)
+*(.text._ZN5Bosch6BMI0889Gyroscope16BMI088_Gyroscope13RegisterCheckERKNS2_17register_config_tE)
+*(.text._ZN5Bosch6BMI0889Gyroscope16BMI088_Gyroscope13RegisterWriteENS1_8RegisterEh)
```

Comment:
Is there any tooling to help keep this maintained?

Reply:
CI keeps track of it and throws an error if it mismatches.
https://github.com/PX4/PX4-Autopilot/actions/runs/19111506804/job/54609546150